programs/locker/src/state/vesting_escrow.rs

```diff
@@ -110,13 +134,32 @@ impl VestingEscrow {
     pub fn update_recipient(&mut self, new_recipient: Pubkey) {
         self.recipient = new_recipient;
     }
+
+    pub fn validate_cancel_actor(self, signer: Pubkey) -> Result<bool> {
```

Comment:
we should return Result<()>, return error if it doesn't pass

Reply:
fixed